README.md

```diff
@@ -97,6 +97,9 @@ You can set up the credentials, nodeSelector, volume sizes (default volumes crea
 | `timescale-prometheus.enabled`                      | If false Timescale-Prometheus Connector will not be created| `true` |
 | `timescale-prometheus.image`                        | Docker image to use for the Connector                 | `timescale/timescale-prometheus:0.1.0-alpha.2` |
 | `timescale-prometheus.connection.dbName`            | Database to store the metrics in                      | `postgres`  |
+| `timescale-prometheus.connection.user`              | User used for connection to db | `postgres` |
```

Comment:
Even though these are the defaults, they are added here, for easier discoverability by users